site/content/contribute/desktop/developer-setup/arch.md

```diff
@@ -0,0 +1,16 @@
+1. Open Terminal
```

Comment:
does this mean we support arch? if we are not officially supporting it should be added as the first line. I like having the process documented, though

Reply:
I actually added this since we had a developer setup page for Arch for the server repo, so I figured since it's a popular distro it would make sense to include it. 0/5 on if we keep it or not, but no I don't think we officially support Arch though.

Reply:
I'm totally ok with having the page, but if it is not supported we should say so.

Reply:
Thing is I know we don't officially support the Desktop App for *use* on Ubuntu, but for development I don't know if we have a *supported* list of distributions :S

Reply:
Docs recently removed Archlinux Docker install steps on a recommendation from Support citing that we don't officially support Archlinux.

Reply:
Okay, should we remove this outright then? Or just do what @Willyfrog suggested?

Reply:
Given that our docs (dev, product, and api) are our official sources of product truth, I recommend removing content that isn't officially supported. If it's in the docs, it looks like it's supported, and can result in a Support inquiry. However, that being said, I'm not clear on what distributions we officially support from a development-specific perspective, so deferring to Engineering on this particular decision.

As a related aside, we're migrating unofficial product docs content to the User Forum where the community can maintain as needed.

Reply:
The biggest reason I've included documentation for Arch here is that it's a popular distributions that many others are based on, similar to how there are many based on Ubuntu and some based on Red Hat. I feel like it covers a lot of bases for us and expands our connection with the open-source community.

1/5 on whether we keep it or not, but I think there's some value in it.

Reply:
I'm 5/5 for clearly denoting that it's not officially supported, if the decision is made to keep it in. :)